ferrilata-devilline/Models/Pitch.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+namespace ferrilata_devilline.Models
+{
+    public class Pitch
```

Comment:
this is probably a data transfer object and pitches in the database probably have IDs, maybe approval stamps etc, so i'd put it in a folder for DTOs

Reply:
DTO sounds like something that we will need at a later stage. For this user story I don't see the need.